docs/reference/migration/migrate_8_0/snapshots.asciidoc

```diff
@@ -28,3 +28,15 @@ This change will affect both newly created repositories and existing repositorie
 explicitly specified.
 
 For more information on the compress option, see <<modules-snapshots>>
+
+[float]
+==== The S3 repository plugin uses the DNS style access pattern by default
+
+Starting in version 7.3 using the path style access pattern with the S3 repository is deprecated.
```

Comment:
I'm not sure we should deprecate this at this time. AWS's SDK is going to support path-style access for a very long time to come (even after Sept. 2020, given that the SDK will still be used to connect to older repos).

Reply:
Same point as in the above, why make a breaking change then (that we know for a fact will break some environments)?

Reply:
The focus of this section should be on the fact that the behavior changes between 7.2-  and 7.3+, going back to the 6.x behavior, and that the 7.0-7.2 behavior can be restored by setting this new setting. The deprecation of that new setting is secondary. As said, given that we don't have a time frame for removing this setting, I would prefer to defer the deprecation to a later time.